pkg/yurtctl/constants/constants.go

```diff
@@ -199,5 +199,43 @@ spec:
               fieldPath: spec.nodeName
         - name: STATIC_POD_PATH
           value: {{.pod_manifest_path}}
+`
+	RevertServantJobTemplate = `
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{.jobName}}
+  namespace: kube-system
+spec:
+  template:
+    spec:
+      hostPID: true
+      hostNetwork: true
+      restartPolicy: OnFailure
+      nodeName: {{.nodeName}}
+      volumes:
+      - name: host-var-tmp
+        hostPath:
+          path: /var/tmp
+          type: Directory
+      containers:
+      - name: yurtctl-servant
+        image: {{.yurtctl_servant_image}}
+        imagePullPolicy: Always
+        command:
+        - /bin/sh
+        - -c
+        args:
+        - "nsenter -t 1 -m -u -n -i /bin/yurtctl revert edgenode"
+        securityContext:
+          privileged: true
+        volumeMounts:
+        - mountPath: /tmp
+          name: host-var-tmp
+        env:
+        - name: NODE_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: spec.nodeName
 `
```

Comment:
env STATIC_POD_PATH is missed

Reply:
`yurtctl convert` provides the value of STATIC_POD_PATH：

if err = kubeutil.RunServantJobs(co.clientSet, map[string]string{
		"provider":              string(co.Provider),
		"action":                "convert",
		"yurtctl_servant_image": co.YurctlServantImage,
		"yurthub_image":         co.YurhubImage,
		"joinToken":             joinToken,
		"pod_manifest_path":     co.PodMainfestPath,
	}, edgeNodeNames, true);

however, `yurtctl revert` not provides it:

if err = kubeutil.RunServantJobs(ro.clientSet,
		map[string]string{
			"action":                "revert",
			"yurtctl_servant_image": ro.YurtctlServantImage,
		},
		edgeNodeNames, false); 

so I delete the env STATIC_POD_PATH in `RevertServantJobTemplate`

Reply:
yeah, must had missed STATIC_POD_PATH var for `yurtctl revert` command, how ablout add STATIC_POD_PATH for `yurtctl revert` cmd?

Reply:
OK.